core/json/src/main/scala/net/liftweb/json/JsonAST.scala

```diff
@@ -19,32 +19,97 @@ package json
 
 import scala.language.implicitConversions
 
+/**
+ * This object contains the abstract snytax tree (or AST) for working with JSON objects in lift-json.
+ *
+ * The purpose of the JsonAST is to represent and manipulate JSON by leveraging Scala language features like types,
+ * case classes, etc. The AST should allow you to represent anything you could imagine from JSON land using the Scala
+ * type system.
+ *
+ * Everything in the AST has a single root: JValue. A JValue could, quite literally, be anything. It could be an
+ * an object (represented by [[JObject]]), a string ([[JString]]), a null ([[JNull]]), and so on. So, when constructing
+ * a JSON object with the AST directly you might construct something like the following:
+ *
+ * {{{
+ * JObject(JField("bacon", JBool(true)) :: JField("spinach", JBool(false)))
+ * }}}
+ *
+ * Once serialized to the string representation of JSON you would end up with the following:
+ *
+ * {{{
+ * {
+ *   "bacon":true,
+ *   "spianch":false
+ * }
+ * }}}
+ */
 object JsonAST {
   import scala.text.{Document, DocText}
   import scala.text.Document._
 
-  /** Concatenates a sequence of <code>JValue</code>s.
-    * <p>
-    * Example:<pre>
+  /**
+    * Concatenate a sequence of `JValue`s together.
+    *
+    * This would be useful in the event that you have a handful of `JValue` instances that need to be
+    * smacked together into one unit.
```

Comment:
It's not really for one “unit” generally, but specifically to combine a set of values into a JSON array, right?